Examples/Python/python/acts/examples/reconstruction.py

```diff
@@ -123,10 +123,44 @@
         "maxSharedHits",
         "maxChi2",
         "nMeasurementsGroupMin",
+        "requireReferenceSurface",
     ],
-    defaults=[(None, None)] * 7 + [None] * 7,
+    defaults=[(None, None)] * 7 + [None] * 8,
 )
 
+
+def trackSelectorDefaultKWArgs(c):
+    """
+    Encapsulate this boilerplate code into a function so different uses do not get out of sync
+    """
+    return {
+        **acts.examples.defaultKWArgs(
```

Comment:
nitpick: I think this should give the same?
```suggestion
    return acts.examples.defaultKWArgs(
```